white-test-app-backend/src/main/java/pl/lodz/p/white/whitetestapp/accountmanager/manager/AccountManager.java

```diff
@@ -51,4 +56,25 @@ public void deleteRedactor(Account account) throws WrongRequestException {
     public List<Account> getAllRedactors() {
         return repository.findAllByRole(Role.REDACTOR);
     }
+
+    @Override
+    public void addRedactor(AccountDto account) throws EntityAlreadyExistsException, ConstraintViolationException, IllegalArgumentException {
+        try {
+            Account daoCheck = findOne(account.getUsername());
+            throw new EntityAlreadyExistsException();
+        } catch (EntityNotFoundException e) {
+            createRedactorEntity(account);
+        }
+    }
+
+    private void createRedactorEntity(AccountDto account) throws ConstraintViolationException, IllegalArgumentException  {
+        Account newUserEntity = new Account();
+        newUserEntity.setUsername(account.getUsername());
```

Comment:
mozna przerzucić do klasy(np. `AccountMapper`) jako statyczna metodę. I jak poprawisz settery/gettery to:
`return new  Account()
        .newUserEntity.setUsername(account.getUsername())
        .newUserEntity.setEmail(account.getEmail());
......
`
tak jak w innych mapperach